jupyterhub/jupyterhub/base/jupyterhub-dc.yaml

```diff
@@ -98,6 +98,17 @@ spec:
         volumeMounts:
         - mountPath: /opt/app-root/configs
           name: config
+        readinessProbe:
+            tcpSocket:
+              port: 8080
```

Comment:
can you change this to port 8081? @lucferbux

Reply:
In a subsequent MR we will be getting rid of the configurableHTTPproxy at port 8080